content/documentation/coffeeEditorNg/content.md

```diff
@@ -1,35 +1,45 @@
 +++
 fragment = "content"
-title = "Coffee Editor NG"
+title = "Modeling Tool Approaches"
 weight = 130
 [sidebar]
   sticky = true
 +++
 
-The Coffee Editor NG is a comprehensive example modeling tool based on EMF Cloud technologies and can also act as an architecture blueprint for your custom modeling tool.
-It thus brings together a set of best practices in architecture and technology selection from both inside and outside of EMF Cloud.
-This full-featured example tool is written entirely in Typescript and includes a central Model Hub serving a sample modeling language to a variety of editors, such as a diagram editor, a form-based editor, as well as a textual DSL editor.
+EMF Cloud provides two complementary approaches for building modeling tools. Each approach is tailored to specific requirements and is integrated as a core part of the framework.
 
 <div style="text-align:center; margin-bottom:20px">
-  <img src="../../images/coffeeeditormodelhub.svg" alt="Overview of the Model Hub for the Coffee Editor NG" width="70%" />
+  <img src="../../images/modelingtools.svg" alt="Overview of the Model Hub" width="70%" />
 </div>
 
-The core of the Coffee Editor NG is a model language contribution to the Model Hub.
-With this language contribution, we register the modeling language alongside the capabilities for handling this modeling language in the Model Hub.
-The contributed Coffee Editor NG Model is defined with Langium, an open-source language toolkit for textual languages.
-The Langium-based language is then integrated into the Model Hub API with the generic *EMF Cloud Model Hub Langium* integration library, so that Model Hub clients can interact with the language on model level and benefit from Langium's efficient persistence, cross-reference management and validation mechanism.
+## Model Hub Approach
 
-In order to also simplify the editing capabilities, including undo and redo, on model level -- rather than on text level -- we use the *EMF Cloud Editing Domain*, which provies state management, as well as a command API and a command stack for arbitrary JSON models.
+This approach focuses on comprehensive model management. It is built around a central Model Hub that offers robust capabilities such as command-based editing, state management with undo/redo, and persistence. This is ideal for scenarios where a powerful environment for managing complex models is required. Detailed examples for this approach can be found in the Model Hub repository:
 
-Finally, the Coffee Editor NG language contribution adds a custom coffee-model-specific API to be used by clients in order to provide reusable model queries and manipulation functions to the clients.
+https://github.com/eclipse-emfcloud/modelhub/tree/main/examples/theia
```

Comment:
I think it'd be best to link here to separate EMF Cloud Getting Starting pages (or dedicated sub-sections in a common Getting Started page), which give one example or a high-level overview on what the two repositories and their examples look like and have multiple outgoing links to the respective repositories.